book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
```

Comment:
- [ ] Kindly add a method for adding rentals to achieve the following relation `A book has many rentals`. The method will be something like this:
```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
```